Cargo.toml

```
@@ -7,6 +7,7 @@ members = [
     "spectrum-crypto",
     "spectrum-vrf",
     "spectrum-move",
+    "spectrum-data-bridge",
```

Let's keep connectors for each chain in separate crates
```suggestion
    "spectrum-ergo-connector",
```